LICENSE

```diff
@@ -217,6 +217,8 @@ The text of each license is the standard Apache 2.0 license.
 
    proto files from cncf/udpa: https://github.com/cncf/udpa Apache 2.0
    proto files from envoyproxy/data-plane-api: https://github.com/envoyproxy/data-plane-api  Apache 2.0
+   proto files from satellite/envoy/: https://github.com/envoyproxy/envoy  Apache 2.0
```

Comment:
I think this is skywalking's als fork? @mrproliu 
If so, this is not from envoy. And I can see we have envoy data panel proto licences already.

Reply:
I checked, this is a shell of envoy ALS, and it belongs to SkyWalking.

Reply:
```suggestion
```